### PR TITLE
Pass environment to docker

### PR DESCRIPTION
**Pull request by ab77**
Change-type: patch

### CODE REVIEW

**Comment by ab77** [comment]
@balena-ci I self-certify!

File: automation/jenkins_generate_ami.sh [hunk 1]
```diff
@@ -42,12 +43,12 @@ YOCTO_IMAGES_PATH="${WORKSPACE}/build/tmp/deploy/images/${MACHINE}"
...
 
 ORIG_IMAGE="${YOCTO_IMAGES_PATH}/${IMAGE_NAME}"
-PRELOADED_IMAGE="$(mktemp -p ${YOCTO_IMAGES_PATH})"
+PRELOADED_IMAGE="$(mktemp -p "${YOCTO_IMAGES_PATH}")"
 
 cp "${ORIG_IMAGE}" "${PRELOADED_IMAGE}"
 
 # AMI names must be between 3 and 128 characters long, and may contain letters, numbers, '(', ')', '.', '-', '/' and '_'
-VERSION=$(cat "${YOCTO_IMAGES_PATH}/VERSION_HOSTOS" | sed 's/+/-/g')
+VERSION=$(cat < "${YOCTO_IMAGES_PATH}/VERSION_HOSTOS" | sed 's/+/-/g')
```
Comment (klutchell) [inline automation/jenkins_generate_ami.sh:51]: Why the extra redirect here?

Comment (ab77) [reply automation/jenkins_generate_ami.sh:51]: https://github.com/koalaman/shellcheck/wiki/SC2002

```
In automation/jenkins_generate_ami.sh line 50:                                                                                                        
VERSION=$(cat "${YOCTO_IMAGES_PATH}/VERSION_HOSTOS" | sed 's/+/-/g')                                                                                  
              ^-- SC2002 (style): Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.                                                 
```

Comment (klutchell) [reply automation/jenkins_generate_ami.sh:51]: I think the proper way to address that warning is to remove the cat altogether.
```suggestion
VERSION="$(sed 's/+/-/g' "${YOCTO_IMAGES_PATH}/VERSION_HOSTOS")"
```

Comment (ab77) [reply automation/jenkins_generate_ami.sh:51]: both are valid, but yes, the later is more concise.

**Pull request merged**